packages/atomic/src/components/atomic-layout/atomic-layout.pcss

```diff
@@ -0,0 +1,27 @@
+@import './search-layout.pcss';
+
+atomic-layout {
+  width: 100%;
+  height: 100%;
+  display: grid;
+}
+
+atomic-layout-section[section='search-box'] {
```

Comment:
Although I like the `section=` approach too, I would prefer if we used `atomic-layout-section-search-box` and so on instead, since this is the pattern we use for result template sections.

Reply:
result template ones are more opinionated and have different features/behaviour. Here it's only to identify a section with common/generic options. It would be a lot more components for few benefits.